src/Calendar.js

```diff
@@ -205,27 +205,31 @@ const __class = declare('argos.Calendar', [ _Widget, _ActionMixin, _Templated],
     this.date.selectedDateMoment = null;
   },
   createMonthModal: function createMonthModal() {
-    this._monthModal = new Modal({ id: 'month-modal ' + this.id, showBackdrop: false, positioning: 'center', closeAction: 'hideMonthModal', actionScope: this });
-    if (this.domNode.offsetParent) {
-      this._monthModal.placeModal(this.domNode.offsetParent);
-    } else {
-      this._monthModal.placeModal(this.domNode);
+    if (!this._monthModal) {
+      this._monthModal = new Modal({ id: 'month-modal ' + this.id, showBackdrop: false, positioning: 'center', closeAction: 'hideMonthModal', actionScope: this });
+      if (this.domNode.offsetParent) {
+        this._monthModal.placeModal(this.domNode.offsetParent);
```

Comment:
Duplicate code in each block. This can be compressed down to `this._monthModal.placeModal(this.domNode.offsetParent || this.domNode);` with no if statement.